projects/ngx-datatable/src/lib/components/body/summary/summary.harness.ts

```diff
@@ -0,0 +1,12 @@
+import { ComponentHarness } from '@angular/cdk/testing';
+
+export class SummaryHarness extends ComponentHarness {
+  static readonly hostSelector = 'datatable-summary-row';
+
+  private summaryRowCells = this.locatorForAll('datatable-body-cell');
+
+  async getSummaryRowCell(index: number) {
+    const cells = await this.summaryRowCells();
+    return cells[index];
```

Comment:
The TestElement should ideally stay internal. I guess you can just call `text()`